gitpod-io/gitpod-vscode-desktop
# Use createTelemetryLogger api

## Description

Use createTelemetryLogger api

Related https://linear.app/gitpod/issue/IDE-76/

## How to test

- [x] Check events and error reporting works as expected

You can check telemetry events are logged in the `Extension telemetry` output channel (needs vscode log level set to trace)
![image](https://github.com/gitpod-io/gitpod-vscode-desktop/assets/25115070/3b06d6f6-e601-4ba0-8e75-2507c2ba47c9)
